cypress/support/pages/signIn.pageObject.js

```diff
@@ -25,6 +25,15 @@ class SignInPageObject extends PageObject {
       .type(password);
   }
 
+  fillLoginCredentials(email, password) {
```

Comment:
The method name 'fillLoginCredentials' is not a verb or a verb phrase as per the coding standards. Consider renaming the method to a verb or verb phrase. For example, 'enterLoginCredentials' or 'inputLoginCredentials' could be more appropriate.

Reply:
Done